scene/main/canvas_layer.cpp

```diff
@@ -67,6 +67,14 @@ void CanvasLayer::set_visible(bool p_visible) {
 	}
 }
 
+void CanvasLayer::show() {
+	set_visible(true);
+}
+
+void CanvasLayer::hide() {
+	set_visible(false);
+}
```

Comment:
You need to also bind the methods in `_bind_methods`, otherwise they're not part of the scripting API.

Reply:
Sorry. I got lazy and did not check other places before creating the PR. Fix sent.